benchmark_runner/common/template_operations/templates/vdbench/internal_data/vdbench_pod_template.yaml

```diff
@@ -63,6 +63,10 @@ metadata:
   name: vdbench-{{ kind }}-{{ trunc_uuid }}-{{ scale }}
   {%- endif %}
   namespace: {{ namespace }}
+  annotations:
```

Comment:
Please put if before annotations

Reply:
I agree; an empty annotations list does nothing.

However, it might not be a bad idea to put this in a data template in a way that can be reused with other workloads.  j2 being j2, though, that's not always very simple.

Reply:
@RobertKrawitz  I'm not sure exactly what you are suggesting here, but IMHO the "annotation" key-value should be in any pod/VM YAML it's the correct way, same as the "label" key-value for convenience and ease of use, especially when doing some manual intervention for investigations.